llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2963,6 +2965,54 @@ class VPScalarIVStepsRecipe : public VPRecipeWithIRFlags {
   }
 };
 
+/// A recipe to represent scalar stores that sink outside the vector loop.
```

Comment:
Would be good to remove the reference of sinking, the main thing is it should be outside any vector loop region an will only generate a single instance